pool_automation/roles/indy_repo/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+# TODO it might make sense to split the role into indy_node_repo and indy_sdk_repo
```

Comment:
Also it might make sense to move HTTPS support and adding keys to some common role, and put remaining `apt_repository` tasks in corresponding roles (indy_node, indy_cli)

Reply:
yes, splitting the code will definitely lead to such changes

Reply:
on the other hand it might not: https and the key are common for both repos only for the current moment but things might change since they are actually separate repos